target/linux/meson/patches-5.15/903-Support-for-EMMC.patch

```diff
@@ -0,0 +1,58 @@
+--- a/arch/arm/boot/dts/Makefile
++++ b/arch/arm/boot/dts/Makefile
+@@ -365,6 +365,7 @@ dtb-$(CONFIG_MACH_MESON8) += \
+ 	meson8b-ec100.dtb \
+ 	meson8b-mxq.dtb \
+ 	meson8b-odroidc1.dtb \
++	meson8b-onecloud.dtb \
+ 	meson8m2-mxiii-plus.dtb
+ dtb-$(CONFIG_ARCH_MMP) += \
+ 	pxa168-aspenite.dtb \
+--- a/drivers/mmc/core/mmc.c
++++ b/drivers/mmc/core/mmc.c
```

Comment:
why this is required, add a patch header to describe it, and you should submit this to upstream kernel

this patch should be numbered as 500